### PR TITLE
docs: add v2018.1.2 release notes

### PR DESCRIPTION
**Pull request by mweinelt**
As discussed on the last mumble we want to do another maintenance release for v2018.1.x. The release date is not set in stone and we can still merge fixes into v2018.1.x, although I'm not expecting anything in particular. Surprise me! :wink:

### CODE REVIEW

File: docs/releases/v2018.1.2.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  (`#1530 <https://github.com/freifunk-gluon/gluon/issues/1530>`_)
+
+  As the path to both config mode and status page were changed between versions
+  users could be affected by a redirect to a no more valid URL. 
+
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to v4.4.153
```
Comment (rotanid) [inline docs/releases/v2018.1.2.rst]: maybe add something about several bugfixes for batman-adv here, too? 0bb3742f51d1af0ebc1394c72e899524b0adf81d

**Comment by rotanid** [comment]
i don't see any reason to wait long with this release, we actually wanted to do it in october and there are no open issues where we expect a fix soon

File: docs/releases/v2018.1.2.rst (hunk 1 above)
Comment (mweinelt) [reply docs/releases/v2018.1.2.rst]: Added a reference to the commit in openwrt-routing, thanks!

File: docs/releases/v2018.1.2.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* batman-adv has received two bugfixes, which where (`backported <https://github.com/openwrt-routing/packages/commit/7bf62cc8b556b5046f9bbd37687376fe9ea175bb>`) from v2018.4
```
Comment (NeoRaider) [inline docs/releases/v2018.1.2.rst]: Remove the parentheses, add an understore.

**Comment by NeoRaider** [comment]
The release notes are missing a link in the index, otherwise LGTM.

**Comment by mweinelt** [comment]
Done

File: docs/releases/v2018.1.2.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  As the path to both config mode and status page were changed between versions
+  users could be affected by a redirect to a no more valid URL. 
+
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to v4.4.153
+* batman-adv has received two bugfixes, which were `backported <https://github.com/openwrt-routing/packages/commit/7bf62cc8b556b5046f9bbd37687376fe9ea175bb>`_ from v2018.4
```
Comment (NeoRaider) [inline docs/releases/v2018.1.2.rst]: One remaining nitpick: I guess this should be listed under "Bugfixes" rather than "Other changes"?

**Pull request merged**